### PR TITLE
New styling for add-to/create list forms

### PR DESCRIPTION
**Pull request by phamann**
As discussed with Paul, this adds an acceptable level of styling to the overlay forms to create or add content to list.

#### Mobile
google chrome

#### Desktop
google chrome

### CODE REVIEW

File: client/components/lists/main.scss [hunk 1]
```diff
@@ -78,10 +78,76 @@
...
 	@include oGridRespondTo(M) {
 		margin-bottom: 3px;
 	}
 
-	margin-left: 10px;
+}
+
+.myft-list-form__wrapper {
+	@include oGridRespondTo(M) {
+		min-width: 640px;
```
Comment (phamann) [inline client/components/lists/main.scss]: I chose this at random. Would be nice to be able to pull in an `o-grid` column value. Is this possible via a mixin/helper?

Comment (laurieboyes) [inline client/components/lists/main.scss]: // Default layouts
$o-grid-layouts: (
    S:  490px,
    M:  740px,
    L:  980px,
    XL: 1220px,
);

http://registry.origami.ft.com/components/o-grid about 3 quarters down the page

**Comment by laurieboyes** [comment]
Overlay after article saved (as opposed to copying from list to list) needs a look

**Comment by laurieboyes** [comment]
Core journey needs some vertical margins
screen shot 2015-12-07 at 10 43 48

**Comment by keirog** [comment]
Shouldn't the button text be "Copy"?

**Comment by phamann** [comment]
New core experience styling:

google chrome

**Pull request merged**